pasqal_cloud/client.py

```diff
@@ -160,10 +199,32 @@ def _send_batch(self, batch_data: Dict[str, Any]) -> Dict[str, Any]:
 
     def _get_batch(self, batch_id: str) -> Dict[str, Any]:
         response: Dict[str, Any] = self._request(
-            "GET", f"{self.endpoints.core}/api/v1/batches/{batch_id}"
+            "GET", f"{self.endpoints.core}/api/v2/batches/{batch_id}"
         )["data"]
         return response
 
+    def _get_batch_jobs(self, batch_id: str) -> list[Dict[str, Any]]:
+        return self._request_with_pagination(
+            "GET",
+            f"{self.endpoints.core}/api/v2/jobs",
+            params={
+                "batch_id": batch_id,
+                "order_by": "ordered_id",
+                "order_by_direction": "ASC",
```

Comment:
These order-by parameters are required to ensure that the jobs appear in the same order as they do in the GET /batches/{id} response. 

I am hesitant to set this as the default order-by value on core-api

Reply:
What's the current default ordering? If this can help keeping jobs in the same order, I'm all for it

Reply:
It is, from what I see:

order_by= "created_at"
order_by_direction: "DESC"

Reply:
The abstract repository class applies default ordering by `id` and `created_at`. Since this ordering is used for all endpoints, I don't think it's necessary to change it specifically for jobs. It's better to maintain consistent default ordering across all endpoints. We can still change the ordering when calling the endpoint, as done here.

Reply:
>These order-by parameters are required to ensure that the jobs appear in the same order as they do in the GET /batches/{id} response.

Can you add this as a comment in the code so that we know why we define an explicit ordering